content/spin/v2/install.md

```diff
@@ -286,10 +286,66 @@ Then install plugins by name.
 
 Python:
 
+You may have see a `py2wasm` plugin in your travels. Please note, `py2wasm` (an experiment to build a Spin Python SDK using CPython, Wizer, and PyO3) has since been replaced by Componentize-Py. The process of getting your system ready to write Wasm-powered Python applications is as follows:
+
+<!-- @selectiveCpy -->
+
+```bash
```

Comment:
Thanks, I have created a callout box to mention the historic side of things.
We could make this a lot shorter by just giving the user the dependencies to install, but I think that in the long run creating an app using the template and then running the pip install requirements.txt file (that the template provides in the new app) is more robust.

Reply:
Yep I think that's fine, I'm just doubtful it should be in a section called "Installing Templates and Plugins."  There's no system level installation as there was with py2wasm: you install the stuff _for each individual application_ as you venv and build it.  I'd suggest moving it to "Compiling Applications" maybe?  Because they will need to set it up for each app they create, not just once, and not at Spin install time.  Hope that makes more sense of where I'm coming from...!